WordPress-Core/ruleset.xml

```diff
@@ -235,6 +245,10 @@
 		<severity>0</severity>
 	</rule>
 
+	<!-- Check correct spacing of language constructs. This also ensures that the
+	     above rule for not using brackets with require is fixed correctly.
+		 https://github.com/WordPress/WordPress-Coding-Standards/issues/1153 -->
+	<rule ref="Squiz.WhiteSpace.LanguageConstructSpacing"/>
```

Comment:
Note that this sniff is deprecated and [removed in PHPCS 4.0](https://github.com/squizlabs/PHP_CodeSniffer/blob/077ebabafe34c7bc3c214bc8c9f591580707914b/CHANGELOG.md#L43-L44), but fixing it should be done in a different PR.

Reply:
Issue #1616 is already open about that and as soon as we decide to up the minimum supported PHPCS version to 3.4.1 or higher, it should be swopped out.